src/components/WalletAssetDetails.vue

```diff
@@ -287,10 +287,12 @@ export default class WalletAssetDetails extends Mixins(FormattedAmountMixin, Cop
     return asset && asset.symbol === KnownSymbols.XOR;
   }
 
-  get isCleanHistoryDisabled(): boolean {
-    return !this.asset
-      ? true
-      : !this.activity.filter((item) => [item.assetAddress, item.asset2Address].includes(this.asset.address)).length;
+  get isCleanHistoryEnabled(): boolean {
+    if (!this.asset) return false;
```

Comment:
Could we have history items without asset at all (such as Set Referral) and should we have ability to remove the history?
Also as I understand, history removing right now is disabled, so maybe we should return to it later and rework removing functionality.

Reply:
We could have history items without asset. Earlier this clean history button was shown in history filtered by asset (asset details view). I just updated this place because the data structure of activity was updated from array to object. 
After this PR we will store only unsynced with subquery history in localstorage. So maybe we don't need to have this functionality now